src/executor/insert.rs

```diff
@@ -116,7 +116,7 @@ where
     {
         // so that self is dropped before entering await
         let mut query = self.query;
-        if db.support_returning() && <A::Entity as EntityTrait>::PrimaryKey::iter().count() > 0 {
```

Comment:
I believe all our primary keys has len > 0, so this check is redundant?

Reply:
Redundant indeed!